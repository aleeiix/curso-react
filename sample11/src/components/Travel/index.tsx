const Travel = () => {
    return (
        <div className="card text-center" style={{width: "20rem"}}>
            <i className="material-icons" style={{fontSize: "5rem"}}>Airplane Mode Active</i>
            <h4>Travel Anywhere</h4>
        </div>
    )
}

export default Travel
