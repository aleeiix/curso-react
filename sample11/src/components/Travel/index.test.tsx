import { render, screen } from "@testing-library/react";

import Travel from "./index";

describe("Travel Component", () => {
  it("Display text component as snapshot", () => {
    const { container } = render(<Travel />);

    expect(container).toMatchInlineSnapshot(`
      <div>
        <div
          class="card text-center"
          style="width: 20rem;"
        >
          <i
            class="material-icons"
            style="font-size: 5rem;"
          >
            Airplane Mode Active
          </i>
          <h4>
            Travel Anywhere
          </h4>
        </div>
      </div>
    `);
  });

  it("Display header and paragraph", () => {
    render(<Travel />);
    const header = screen.getByRole("heading", { name: /travel anywhere/i });

    // MUESTRA CONSOLE LOG DEL TEMPLATE QUE HAY RENDERIZADO
    // screen.debug();
  });
});
