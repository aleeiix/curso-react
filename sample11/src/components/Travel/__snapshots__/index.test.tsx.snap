// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Travel Component Display text component as snapshot 1`] = `
<div>
  <div
    class="card text-center"
    style="width: 20rem;"
  >
    <i
      class="material-icons"
      style="font-size: 5rem;"
    >
      Airplane Mode Active
    </i>
    <h4>
      Travel Anywhere
    </h4>
  </div>
</div>
`;
